src/solvers/td3.ml

```diff
@@ -1036,6 +1062,13 @@ module WP =
           )
 
         let finalize ~vh ~reachable =
+          (* replay superstable messages from unknowns that are still reachable *)
+          if incr_verify then (
+            HM.iter (fun (l:S.v) m ->
+                Messages.add m
+              ) var_messages;
+          );
+
```

Comment:
Does everything still work if these are also moved from `finalize` back to `init`? There's another such change in `IncrWrite`.